.github/workflows/Operator-CI.yaml

```diff
@@ -75,13 +75,16 @@ jobs:
           DEV_EVENT_BRIDGE_SSO_SECRET: ${{ secrets.DEV_EVENT_BRIDGE_SSO_SECRET }}
         run: |
           kubectl create namespace $NAMESPACE
+          # Create addon-smart-events-operator-parameters secret
+          kubectl create secret generic addon-smart-events-operator-parameters \
+          --from-literal=EVENT_BRIDGE_SSO_URL=https://sso.redhat.com/auth/realms/redhat-external \
+          --from-literal=EVENT_BRIDGE_SSO_CLIENT_ID=$DEV_EVENT_BRIDGE_SSO_CLIENT_ID \
+          --from-literal=EVENT_BRIDGE_SSO_SECRET=$DEV_EVENT_BRIDGE_SSO_SECRET \
+          -n $NAMESPACE
           # Deploy operator
           kubectl apply -f shard-operator/target/kubernetes/bridgeexecutors.com.redhat.service.bridge-v1.yml
           kubectl apply -f shard-operator/target/kubernetes/bridgeingresses.com.redhat.service.bridge-v1.yml
-          kubectl apply -f shard-operator/target/kubernetes/kubernetes.yml
-          kubectl set env deployment/shard-operator EVENT_BRIDGE_SSO_URL=https://sso.redhat.com/auth/realms/redhat-external -n $NAMESPACE
-          kubectl set env deployment/shard-operator EVENT_BRIDGE_SSO_CLIENT_ID=$DEV_EVENT_BRIDGE_SSO_CLIENT_ID -n $NAMESPACE
-          kubectl set env deployment/shard-operator EVENT_BRIDGE_SSO_SECRET=$DEV_EVENT_BRIDGE_SSO_SECRET -n $NAMESPACE
+          kubectl apply -f shard-operator/target/kubernetes/kubernetes.yml          
```

Comment:
Is this `kubernetes.yml` linked to the secret you create few lines above `addon-smart-events-operator-parameters`?

Reply:
yes, it have. https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/MGDOBR-1036/shard-operator/src/main/resources/application.properties#L44